apps/aurora-portal/src/client/routes/_auth/accounts/$accountId/projects/$projectId/compute/-components/Images/-components/CreateImageModal.test.tsx

```diff
@@ -24,32 +23,23 @@ describe("CreateImageModal", () => {
     expect(screen.queryByTestId("dialog")).toBeNull()
   })
 
-  test("calls onClose when Cancel button is clicked", () => {
+  test("calls onClose when Cancel button is clicked", async () => {
     render(<CreateImageModal isOpen={true} onClose={mockOnClose} onCreate={mockOnCreate} />)
 
     const cancelButton = screen.getByText("Cancel")
-    fireEvent.click(cancelButton)
+    await userEvent.click(cancelButton)
```

Comment:
please use `userEvent.setup()` because performing actions like `click` becuse it will not call setup internally in future versions see the recommendation at [this](https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent) section of the documentation. Meaning your test will look like
```
// first setup
const user = userEvent.setup()

// other arrangements

// and then performing action
await user.click(element)
```